### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by gemmen29**
In this project I add the following:

1. Add home and about page mobile-first.
2. Add a desktop version of these pages.
3. Add mobile menu.
4. Make featured section items generated dynamically.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      <a class="nav-bar__mobile-menu-link" href="about.html">About</a>
+    </div>
+    <nav class="h-100 d-flex align-items-center">
+      <a id="mobile-menu-show-btn" class="display-4 text-dark d-md-none"><i class="fas fa-bars"></i></a>
+      <div class="nav-bar__desktop-menu d-none d-md-flex justify-content-end px-5 gap-3 w-100">
+        <a class="nav-bar__link" href="index.html">Home</a>
+        <a class="nav-bar__link active" href="about.html">About</a>
+      </div>
+    </nav>
+  </header>
```
Comment (omar-labana) [inline about.html]: - [x] On the `Home` page and `About` page the header should:
 - - [x]  Contain a hamburger menu on the `left` side on `mobile` screens.
 - - [x]  It's background should be transparent or the same color/pattern as the top section background on `mobile` screens.
 - - [x]  be divided in a top dark bar and a bottom white bar on `desktop` screens.
 - - [x]  The top short dark bar section should contain social media links, logout, and language links aligned to the right on `desktop` screens.
 - - [x] The other bottom bar should contain a logo `to the left` and a list of page links `to the right` on `desktop` screens.
Reference Design
image
image
Your Styling
image
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            <h4 class="display-4 fw-bolder">2021</h4>
+            <h5 class="fs-6 fw-normal">Full-Stack complete course 2021</h5>
+          </div>
+        </li>
+      </ul>
+    </section>
+  </main>
+
+  <footer class="footer">
+    <section class="partners bg-dark text-light pt-4 pb-5">
```
Comment (omar-labana) [inline about.html]: - [x] The `footer` should have more vertical padding to match the design.
- [x] On `desktop` screen size in the `About page` the `footer` should have a dark gray background.
Reference Design
image

File: index.html [hunk 3]
```diff
@@ -1,16 +1,140 @@
...
+        Learn anything you want at any topics in Web, Mobile and Game development. There is also courses in other
+        fields.
+      </p>
+      <h3 class="mb-1 dark-gray">
+        Get professional skill
+      </h3>
+      <p class="dark-gray">
+        30 days money back guarantee and lifetime access
+      </p>
+    </header>
```
Comment (omar-labana) [inline index.html]: - [x] The `header` element shouldn't be used to contain many types of text content it should be used instead as an introduction to a piece of content or section of a webpage., kindly update your `header` tag to a give more semantic meaning.
- [x] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
- [x] Content inside the `hero` section should be aligned a bit to the left on `desktop` screens.
Reference Design
image
Your Styling
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      <h1 class="mt-1 mb-3 red-color">Best Online Courses 2021</h1>
+      <p class="main-header__discreption p-2 mb-2 dark-gray">
+        Our Academy brings the best instructors in all fields who have the experience and passion to help you and teach
+        all kinds of skills.
+      </p>
+      <p class="dark-gray mb-1">
+        Please contact us by Email for any further questions about our academy
+      </p>
+      <a class="main-header__account-link" href="#">bestcourses@online.org</a>
+    </header>
```
Comment (omar-labana) [inline about.html]: - [x] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
Reference Design
image
Your Styling
image

File: index.html [hunk 5]
```diff
@@ -1,16 +1,140 @@
...
+      <a href="#" class="join-button">Join our academy</a>
+    </section>
+    <section class="featured-speakers pb-5">
+      <h2 class="text-center underline mt-5">Featured Instructors</h2>
+      <ul id="speakers"
+        class="speakers col-md-auto mt-5 d-flex flex-md-row flex-column px-md-3 flex-md-wrap">
+      </ul>
+      <a id="more-speakers-botton" class="more-botton d-md-none d-flex">
+        More
+        <div class="down-arrow"></div>
```
Comment (omar-labana) [inline index.html]: - [x] Articles in your mobile screen are a bit uneven in alignment and positioning on `mobile` screen, kindly investigate the issue.
Reference Design
image

Your Styling
image

File: js/index.js [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+      infoDiv.appendChild(speakerInfo);
+      infoDiv.appendChild(speakerDetailedInfo);
+
+      li.appendChild(divSpeakerImage);
+      li.appendChild(infoDiv);
+
+      ul.appendChild(li);
+    });
+  }
+  generateInstructors();
```
Comment (omar-labana) [inline js/index.js:103]: - [x] Please make use of the `window.onload` function to make sure that it works after the page assets are successfully loaded.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @gemmen29, 

Excellent work so far! This is by far one of the best project I have reviewed on this module 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [x] The video presentation is a bit short, I would suggest recording a new one highlighting a piece of code that you find interesting or proud of making along with an overview of the project.

- [x] I noticed some colors are not like the reference design, please use this color scheme `[#272A31, #EC5242, #D3D3D3]`.

- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 6 above)
Comment (gemmen29) [reply js/index.js:103]: Done

File: about.html (hunk 2 above)
Comment (gemmen29) [reply about.html]: Done.

File: about.html (hunk 1 above)
Comment (gemmen29) [reply about.html]: Done.

File: index.html (hunk 3 above)
Comment (gemmen29) [reply index.html]: Done.

File: index.html (hunk 5 above)
Comment (gemmen29) [reply index.html]: Done

File: about.html (hunk 4 above)
Comment (gemmen29) [reply about.html]: Done

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @gemmen29, 

Great work on making the changes requested previously 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [x] The `hero` section on both pages are still not filling the view height, I would suggest using the `min-height` property with a value of `100vh`, I would suggest also omitting the usage of vertical `padding` along with the `!important` keyword (bad practice), you could use a `flex` container instead then add some `gap` between elements.

## Optional suggestions

_N/A_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**